profile/README.md

```diff
@@ -1 +1,9 @@
-RITS is dead. Long live [ARC](https://github.com/UCL-ARC).
+Research IT Services (RITS) was the first team of research software engineers and research computing specialists in University College London.
+
+The team was moved to the [**Centre for Advanced Research Computing**](https://ucl.ac.uk/arc) ([@UCL-ARC](https://github.com/UCL-ARC)) in 2022. 
```

Comment:
2022, right?

Reply:
No 2021. When I started in November 2021 it was already ARC - although my card says RITS.